src/custom/state/claim/hooks/utils.ts

```diff
@@ -27,3 +34,31 @@ export function hasFreeClaim(claims: UserClaims | null): boolean {
 export function transformRepoClaimsToUserClaims(repoClaims: RepoClaims): UserClaims {
   return repoClaims.map((claim) => ({ ...claim, type: ClaimType[claim.type] }))
 }
+
+/**
+ * Helper function to get the repo path for the corresponding network id
+ * Throws when passed an unknown network id
+ */
+export function getClaimsRepoPath(id: number): string {
+  return `${CLAIMS_REPO}${_repoNetworkIdMapping(id)}/`
+}
+
+function _repoNetworkIdMapping(id: number): string {
```

Comment:
should we expect `SupportedChainId` as input?

Reply:
Thanks, I thought we had something like that but I couldn't find it.
Will update